app/views/catalog/_index_header_list_default.html.erb

```diff
@@ -6,17 +6,15 @@
   -%>
   <% document_actions = capture do %>
     <% # bookmark functions for items/docs -%>
-    <%= render_index_doc_actions document, wrapping_class: "index-document-functions col-sm-3 col-lg-2" %>
+    <%= render_index_doc_actions document, wrapping_class: "index-document-functions" %>
   <% end %>
 
-  <h3 class="index_title document-title-heading <%= document_actions.present? ? "col-sm-9 col-lg-10" : "col-md-12" %>">
```

Comment:
These h3s were added as part of an accessibility study.  You change may adversely affect the usability for screen readers.  Is that okay?